str/runners/str_iterative_eh_runner.py

```diff
@@ -45,6 +45,7 @@
 )
 @click.option('--job-memory', help='Memory of the Hail batch job', default='32G')
 @click.option('--job-ncpu', help='Number of CPUs of the Hail batch job', default=8)
+@click.option('--vcf-only', is_flag=True, help='Omits realigned bam and JSON output')
```

Comment:
can we go one step further and have this default to True?

Reply:
d'accord